camelot/view/action_steps/profile.py

```diff
@@ -382,21 +383,22 @@ class EditProfiles(ActionStep):
 
     profiles: List[Profile]
     current_profile: str = ''
-    dialog_class: InitVar(QtWidgets.QDialog) = None
+    dialog_class: QtWidgets.QDialog = None
```

Comment:
die dialog hoeft nt in de action step te zitten, hoeft nt customiseerbaar te zijn

Reply:
Ok, zie b728c4980dd85faef3c20d742b6ae52af8e27fa2